src/extensions/scratch3_microbit/index.js

```diff
@@ -138,6 +134,16 @@ class MicroBit {
          */
         this._busy = false;
 
+        /**
+         * A task queue to track communication tasks sent over the socket.
+         * The bucket in this task queue holds 1 task at a time, and refills
+         * at a rate of 10 tasks per second, from a queue that holds at maximum
+         * 30 tasks.  If more than 30 tasks are added to the task queue in a short
+         * period, some tasks may be rejected (ignored) by the task queue.
+         * @type {TaskQueue}
```

Comment:
Maybe worth adding something about how the TaskQueue is being used to limit the rate of Bluetooth messages by limiting the rate of execution of blocks which send Bluetooth messages.